EssentialCSharp.Web/Migrations/20240925214247_ReferralIdAndCount.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EssentialCSharp.Web.Migrations
+{
+    /// <inheritdoc />
+    public partial class ReferralIdAndCount : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "ReferralCount",
+                table: "AspNetUsers",
+                type: "int",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<string>(
+                name: "ReferrerId",
+                table: "AspNetUsers",
+                type: "nvarchar(max)",
```

Comment:
We probably want to constrain this to some max (256?)

Reply:
What issue does the column being oversized cause? Perf?

Reply:
Notes for future self: https://www.sqlshack.com/varcharmax-data-type-walkthrough-and-its-comparison-with-varcharn-in-sql-server/

Removed this entirely since I am now only storing id on the claims table